README.md

```diff
@@ -0,0 +1,106 @@
+
+Programs in this directory automatically become commands you can use from your terminal.
+
+### What's inside
+
+* **Node.js** 10 with npm 6
+* **Firefox**
+* **Chromium**
+* `chromedriver`, `ffmpeg`, and `xvfb` (for browser tests)
+* JSDuck
+* Debian 9 Jessie (kernel)
+
+### Prerequisites
+
+You'll need to have Docker installed. See [Docker CE for Linux](https://docs.docker.com/install/#server), [Docker for Mac](https://hub.docker.com/editions/community/docker-ce-desktop-mac), or [Docker for Windows](https://docs.docker.com/docker-for-windows/install/).
+
+## Usage
+
+With Fresh installed, navigate your terminal to a project directory in which
+you might need to run commands like `npm install` or `npm test`.
+Before you run such commands, run `fresh-node` to enter a Fresh environment.
+
+```
+you@precious.local:~$ cd dev/oojs
+you@precious.local:oojs$ fresh-node
+
+# fresh: 19.09.0
+# image: docker-registry.wikimedia.org/…/node10-test-browser…
+# os: Debian Linux 9 (Stretch)
+# software: Node.js 10.… (npm 6.…)
+#           Chromium …
+#           Mozilla Firefox …
+#           JSDuck 5.… (Ruby 2.3.3)
+# mount: /oojs ➟ /Users/you/Dev/oojs
+
+🌱  Fresh!
+
+nobody@76010858c836:/oojs$
+```
+
+You can now execute commands such as `npm install`, `npm test` and other `npm run`
+commands.
+
+## How does it work
+
+The first time you start a Fresh environment, Docker will download the
+container image layers from `docker-registry.wikimedia.org`. This may take
+a few minutes.
+
+The fresh-node command uses the [`node10-test-browser` image](./Tutorial.md),
+which is the same exact image used by Jenkins in Wikimedia Foundation CI.
+This means you can trust that if it works in Fresh, it'll work in CI.
+And vice-versa, you can use Fresh to reproduce test failures locally.
+
+### Fast
+
+Fresh is fast. Creating and launching a new container takes only a fraction
+of as second. This is made possible by Docker and its `docker run` command.
+It doesn't need to copy or clone anything. Instead, it references the
+downloaded Docker image as bottom layer (read-only) in an empty container,
+with a new (initially empty) read-write layer on top. This uses the
+"copy-on-write" principle, powered by [UnionFS](https://en.wikipedia.org/wiki/UnionFS).
+
+### Isolation
+
+You can start as many independent Fresh environments as you want,
+without taking up a lot of disk space, and without different sessions
+affecting each other.
+
+Only the directory from which you launch Fresh, is made visbile to the container.
+The rest of your personal hard drive is not mounted, and most networking
+capabilities are also disabled (except toward the Internet). Specifically,
+local servers and networked devices on your home network are not easily
+reachable.
+
+Any changes made elsewhere in the container, are lost once you exit Fresh.
+
```

Comment:
Somewhere around here (Isolation and Why?) I might add a more specific Caveats section just to explicitly call out some dangerous scenarios for which fresh-node won't mitigate, e.g. running vulnerable versions of docker (CVE-2019-5736 et al), accidentally having sensitive data (some config files or tests which contain prod passwords, PII) within a docker-mounted volume, accidentally having dangerous executable code within a docker-mounted volume, etc.